src/components/InfoTip/InfoTip.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from '../Dropdown';
+
+class InfoTip extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+    this.handleBackFocus = this.handleBackFocus.bind(this);
+    this.state = { open: false, footerFocused: false };
+  }
+
+  handleEnterKeyDown(event) {
+    this.setState({ open: !this.state.open });
+    event.preventDefault();
+  }
+
+  handleTabKeyDown(event) {
+    if (this.state.footerFocused) {
+      this.setState({ open: false, footerFocused: false });
+    } else {
+      this.setState({ footerFocused: true });
+    }
+    event.stopPropagation();
+    event.nativeEvent.stopImmediatePropagation();
+  }
+
+  handleKeyDown(event) {
+    if (event.shiftKey && event.keyCode) {
+      return this.handleBackFocus();
+    }
+    switch (event.key) {
+      case 'Enter':
```

Comment:
i think we want to use keyCodes for these instead... and additionally, reference those in `common/helpers.js`... see this [example](https://github.com/patternfly/patternfly-react/pull/190/files#diff-6d029458f6215790b0dcdb4b6585f3c5)

Reply:
this will be used throughout a11y...so nice to go ahead and standardize i think...

Reply:
can you check to see if event.key has better cross browser support than keycode? either way..we can standardize in `helpers.js` for all...

Reply:
Hi @priley86 there isn't this keyCodes in common/helpers, they changed this file in master.